provider/datagen/tests/verify-zero-copy.rs

```diff
@@ -52,8 +52,11 @@ fn main() {
 
     let converter = icu_datagen::create_datagen_provider!(SourceData::default()
         .with_cldr(icu_testdata::paths::cldr_json_root(), "full".to_string())
+        .unwrap()
```

Comment:
Nit, here and below and in datagen.rs: favor `expect` over `unwrap` if you can prove that a condition won't happen, or else handle the error properly

Reply:
This is a test, I don't think we usually do this